components/item-full.js

```diff
@@ -114,6 +114,20 @@ function ItemEmbed ({ item }) {
     )
   }
 
+  if (provider === 'peertube') {
+    return (
+      <div className={styles.videoWrapper}>
+        <div className={styles.videoContainer}>
+          <iframe
+            title='PeerTube Video'
+            allowFullScreen=''
+            src={meta?.href}
```

Comment:
```suggestion
            src={meta?.href}
            sandbox='allow-scripts'
```

See comment regarding `frame-src`.

Reply:
Nice 👍